src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtensionRequiredJavadocPhrase.java

```diff
@@ -0,0 +1,59 @@
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+public class InputDesignForExtensionRequiredJavadocPhrase {
```

Comment:
please rebase on latest code, there will be violation as `/** Config:` become required.